lib/fog/ovirt/requests/compute/v4/create_vm.rb

```diff
@@ -60,7 +91,6 @@ def create_vm(attrs)
             attrs[:memory_policy] = OvirtSDK4::MemoryPolicy.new(:guaranteed => attrs[:memory]) if attrs[:memory].to_i < Fog::Compute::Ovirt::DISK_SIZE_TO_GB
             attrs[:high_availability] = OvirtSDK4::HighAvailability.new(:enabled => attrs[:ha] == "1") if attrs[:ha].present?
 
-            # TODO: handle cloning from template
             process_vm_opts(attrs)
             new_vm = OvirtSDK4::Vm.new(attrs)
             vms_service.add(new_vm)
```

Comment:
Looks like attrs[:clone] should actually be sent here according to: https://github.com/oVirt/ovirt-engine-sdk-ruby/blob/4f1b811ffe81d73d5d5dadd24bc4a3cbaa92d347/sdk/examples/add_independent_vm.rb#L50